osu.Framework/Graphics/Visualisation/VisualisedDrawable.cs

```diff
@@ -298,7 +298,7 @@ void IContainVisualisedDrawables.AddVisualiser(VisualisedDrawable visualiser)
             flow.Add(visualiser);
         }
 
-        void IContainVisualisedDrawables.RemoveVisualiser(VisualisedDrawable visualiser) => flow.Remove(visualiser);
+        void IContainVisualisedDrawables.RemoveVisualiser(VisualisedDrawable visualiser) => flow.Remove(visualiser, true);
```

Comment:
This method is used for transferring parents or changing child ID as well:
https://github.com/ppy/osu-framework/blob/04c5dd65bfb42e48afaa3aab2f9dd0bab28db3b8/osu.Framework/Graphics/Visualisation/VisualisedDrawable.cs#L460-L466
https://github.com/ppy/osu-framework/blob/04c5dd65bfb42e48afaa3aab2f9dd0bab28db3b8/osu.Framework/Graphics/Visualisation/VisualisedDrawable.cs#L287-L288
```suggestion
        void IContainVisualisedDrawables.RemoveVisualiser(VisualisedDrawable visualiser) => flow.Remove(visualiser, false);
```